python-checks/src/test/resources/checks/incorrectExceptionType/incorrectExceptionType.py

```diff
@@ -122,7 +122,7 @@ def raise_with_type_inference():
 def fn_type_inference():
```

Comment:
Maybe you can rename the test now that the FN is fixed!

Reply:
good point, I missed that!